### ISSUE TITLE
robot export not handling oboInOwl:created_by

### ISSUE THREAD

**Issue by turbomam** [ontodev/robot#847]
When I run the following command

```bash
robot export \
--prefix "oboInOwl: http://www.geneontology.org/formats/oboInOwl#" \
--input src/envo/envo-edit.owl \
--header "ID|LABEL|definition|oboInOwl:creation_date|oboInOwl:created_by|SubClass Of"
```

I get a column for `oboInOwl:creation_date` with values and an empty `oboInOwl:created_by` column

`oboInOwl:created_by` annotations are present:

```bash
grep created_by src/envo/envo-edit.owl | wc -l
```

> 752

but they take a variety of object types

- AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_03600022> <0000-0002-6670-9157>)
- AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_08000002> "TermGenie"^^xsd:string)
- AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_2000000> "cjm"^^xsd:string)
- AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_01001569> <http://orcid.org/0000-0002-4366-3088>)

NB @cmungall

**Comment by beckyjackson** [comment]
Hi @turbomam - this should be resolved by #850 now. If you can build the ROBOT JAR from `master`, you should be able to get your exports 😄 

By the way, `oboInOwl` is a built-in prefix in ROBOT, so you don't need to define it with `--prefix`

**Comment by turbomam** [comment]
Sorry, I haven't had time to check this myself yet, but I trust that it works! I'll eventually speak up either way.

Thanks for the predefined prefixes reminder
